client/landing/gutenboarding/components/domain-picker/index.tsx

```diff
@@ -179,7 +179,7 @@ const DomainPicker: FunctionComponent< Props > = ( {
 										isSelected={ currentDomain?.domain_name === suggestion.domain_name }
 										onSelect={ onDomainSelect }
 										key={ suggestion.domain_name }
-										railcarId={ railcarId ? `${ railcarId }${ i + 1 }` : undefined }
+										railcarId={ railcarId ? `${ railcarId }${ i + 1}` : undefined }
```

Comment:
I think that extra space before the close curly was intentional (style)

Reply:
oops, thanks that was accidental deletion

Reply:
I think this is a bug in our current wp-prettier version, and it's probably just best to go with whatever it wants so that the lint CI passes. I think we're just waiting for the fixed version to make it's way through the tubes.